service/src/main/java/uk/gov/hmcts/reform/fpl/model/Child.java

```diff
@@ -32,6 +36,16 @@ public Party toParty() {
         return party;
     }
 
+    @JsonSetter("finalOrderIssued")
+    public void setFinalOrderIssued(String finalOrderIssued) {
+        this.finalOrderIssued = finalOrderIssued;
+    }
+
+    @JsonGetter("finalOrderIssued")
+    public String getFinalOrderIssued() {
+        return this.finalOrderIssued;
+    }
```

Comment:
Shouldn't need these getters and setters, the `@Data` annotation will provide that and seeing as the name json property name matches the variable name you won't need any `@JsonProperty` annotations either

Reply:
Done